### PR TITLE
[AMLII-1957] Fix integrations launcher filesource field creation and logs filepath

### PR DESCRIPTION
**Pull request by soberpeach**
### What does this PR do?

Two small fixes for integrations launcher:
1. Removes the hardcoded name and source for the `fileSource`
2. Corrects the directory for integrations logs files

### Motivation

Fixing broken behavior.

### Possible Drawbacks / Trade-offs

N/A

This isn't in use in production yet, so no changes should be expected.

### Describe how to test/QA your changes

Write a custom integration file like so:

``` yaml
logs:
  - type: integration
    service: integration_service
    source: integration_source
    name: integration_name
```

Check the datadog run directory for the file it creates (in `/opt/datadog/run/integrations/`) on nix machines, and make sure the log file is there. The final path should be `/opt/datadog/run/integrations/integration_service/integration_name.log`

### CODE REVIEW

File: pkg/logs/launchers/integration/launcher.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	ddLog "github.com/DataDog/datadog-agent/pkg/util/log"
 
 	"github.com/DataDog/datadog-agent/comp/logs/agent/config"
-	"github.com/DataDog/datadog-agent/comp/logs/integrations/def"
+	integrations "github.com/DataDog/datadog-agent/comp/logs/integrations/def"
```
Comment (gh123man) [inline pkg/logs/launchers/integration/launcher.go:16]: Is this import used?

File: pkg/logs/launchers/integration/launcher.go [hunk 2]
```diff
@@ -154,10 +154,10 @@ func (s *Launcher) createFile(source *sources.LogSource) (string, error) {
 
 // integrationLogFilePath returns a directory and file to use for an integration log file
 func (s *Launcher) integrationLogFilePath(source sources.LogSource) (string, string) {
-	fileName := source.Name + ".log"
+	fileName := source.Config.Name + ".log"
 	directoryComponents := []string{s.runPath, "integrations", source.Config.Service}
 	directory := strings.Join(directoryComponents, "/")
-	filepath := strings.Join([]string{directory, fileName}, "")
+	filepath := strings.Join([]string{directory, fileName}, "/")
```
Comment (gh123man) [inline pkg/logs/launchers/integration/launcher.go:160]: Can we add a UT to cover this function to prevent such an error in the future?

**Review by gh123man — CHANGES_REQUESTED** [review]

File: pkg/logs/launchers/integration/launcher.go (hunk 2 above)
Comment (soberpeach) [reply pkg/logs/launchers/integration/launcher.go:160]: Added!

**Review by gh123man — APPROVED** [review]

**Pull request merged**